code/modules/mob/living/emote.dm

```diff
@@ -3,6 +3,9 @@
 //Intended to be called by a higher up emote proc if the requested emote isn't in the custom emotes.
 
 /mob/living/emote(var/act, var/m_type=1, var/message = null)
+	if(stat)
+		return

```

Comment:
Since you're adding a stat check here, you can remove the one used specifically for "me" on line 153-156